gammapy/spectrum/sensitivity.py

```diff
@@ -16,8 +16,12 @@ class SensitivityEstimator(object):
 
     Parameters
     ----------
-    irf : `~gammapy.scripts.CTAPerf`
-        IRF object
+    arf : `~gammapy.irf.EffectiveAreaTable`
+        1D effective area
+    rmf : `~gammapy.irf.EnergyDispersion`
+        energy dispersion table
+    bkg : `~gammapy.spectrum.CountsSpectrum`
+        the background array
     livetime : `~astropy.units.Quantity`
         Livetime (object with the units of time), e.g. 5*u.h
     slope : float, optional
```

Comment:
Maybe take and instance of `SpectrumModel` instead? Or at least call it `index`? I think `slope` is nowhere used in Gammapy.

Reply:
Well, only power laws are meaningful and the choice of normalisation units has an impact on the unit of the result. Since index is the only free parameter I just changed slope to index